test/v1alpha1/resources/serving.knative.dev_v1alpha1_knativeserving_cr_1.3.0.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: serving.knative.dev/v1alpha1
```

Comment:
The content of this file is taken from the example we provide when creating KnativeServing CR via console: https://github.com/openshift-knative/serverless-operator/blob/master/olm-catalog/serverless-operator/1.3.0/serverless-operator.v1.3.0.clusterserviceversion.yaml#L5